src/Tizen.Messaging/Tizen.Messaging.Email/EmailSender.cs

```diff
@@ -40,35 +44,39 @@ public static async Task<EmailSendResult> SendAsync(EmailMessage email)
             email.FillHandle();
             email.Save();
 
-            Interop.Email.EmailSentCallback _emailSendingCallback = (IntPtr handle, int result, IntPtr userData) =>
+            int id = _callbackId++;
```

Comment:
you do better to use some lock guard 

```suggestion
            lock(lockObjectForId) {
                  int id = _callbackId++;
                  ...
            }
```